### DISCUSSION TITLE
October Week 3 Release

### DISCUSSION THREAD

**Discussion by khavinshankar** [ohcnetwork/care_fe#6454] [General]
## What's Changed
* Fix Cypress Failure in Assets by @nihal467 in https://github.com/coronasafe/care_fe/pull/6421
* Replaced date-fns utility functions from DateInputV2 component with day.js by @thedevildude in https://github.com/coronasafe/care_fe/pull/6405
* Aligned Service History & Transaction History in Asset Details by @GokulramGHV in https://github.com/coronasafe/care_fe/pull/6382
* Added AMC Warranty filters and expiry labels for assets by @GokulramGHV in https://github.com/coronasafe/care_fe/pull/6364
* whitelisted jwt_token_refresh_interval by @kshitijv256 in https://github.com/coronasafe/care_fe/pull/6418
* Bump uuid from 9.0.0 to 9.0.1 by @dependabot in https://github.com/coronasafe/care_fe/pull/6409
* Using the treeshakable api for echarts by @Omkar76 in https://github.com/coronasafe/care_fe/pull/6424
* Set page title to medicine by @AshrafMd-1 in https://github.com/coronasafe/care_fe/pull/6431
* Track camera feed views and display offline status by @Ashesh3 in https://github.com/coronasafe/care_fe/pull/6408
* Bump react-player from 2.12.0 to 2.13.0 by @dependabot in https://github.com/coronasafe/care_fe/pull/6317
* fixes nutrition tab showing nursing analysis by @rithviknishad in https://github.com/coronasafe/care_fe/pull/6437

## New Contributors
* @thedevildude made their first contribution in https://github.com/coronasafe/care_fe/pull/6405
* @Omkar76 made their first contribution in https://github.com/coronasafe/care_fe/pull/6424
* @AshrafMd-1 made their first contribution in https://github.com/coronasafe/care_fe/pull/6431

**Full Changelog**: https://github.com/coronasafe/care_fe/compare/v23.40.0...v23.41.0

<hr /><em>This discussion was created from the release <a href='https://github.com/coronasafe/care_fe/releases/tag/v23.41.0'>October Week 3 Release</a>.</em>

**Comment by gigincg** [comment]
@bodhish And now we have Service Histories & Statuses for Assets 😄
